Chat/package.json

```diff
@@ -21,7 +21,7 @@
     "@azure/communication-chat": "^1.5.0",
     "@azure/communication-common": "^2.3.1",
     "@azure/communication-identity": "^1.3.0",
-    "@azure/communication-react": "1.16.0",
+    "@azure/communication-react": "1.16.1",
```

Comment:
When we update this, we usually have to copy over changes from server/sample.

Ex.

https://github.com/Azure-Samples/communication-services-web-chat-hero/pull/147/files

I assume because this only a minor bump this wasn't required this time?

Reply:
This is a patch fix for a hot fix that was released. I don't think we need to make any changes to the server side package